### PR TITLE
Cypress add features

### PR DESCRIPTION
**Pull request by monicalopezgris**
Responsiveness & reporter

### CODE REVIEW

**Comment by sjimenez77** [comment]
Hi @monicalopezgris,

I have just run all the tests and got the following errors:

image

Could you please take a look to them and ensure all pass before merging this PR?

Thanks!

File: angular/src/app/user-area/components/login-dialog/login-dialog.component.html [hunk 1]
```diff
@@ -50,7 +50,11 @@
       <mat-checkbox #termsd name="registerTerms">{{'formFields.terms' | transloco}}</mat-checkbox>
 
       <div class="align-right">
+<<<<<<< HEAD
```
Comment (mperezor19) [inline angular/src/app/user-area/components/login-dialog/login-dialog.component.html:53]: Remove this merge text

File: java/mtsj/api/src/main/java/com/devonfw/application/mtsj/imagemanagement/common/api/Image.java [hunk 2]
```diff
@@ -1,5 +1,13 @@
 package com.devonfw.application.mtsj.imagemanagement.common.api;
 
+<<<<<<< HEAD
```
Comment (mperezor19) [inline java/mtsj/api/src/main/java/com/devonfw/application/mtsj/imagemanagement/common/api/Image.java:3]: Same here. Merge comments should be removed

File: java/mtsj/core/src/main/java/com/devonfw/application/mtsj/imagemanagement/common/api/Image.java [hunk 3]
```diff
@@ -1,7 +1,15 @@
 package com.devonfw.application.mtsj.imagemanagement.common.api;
 
+<<<<<<< HEAD
```
Comment (mperezor19) [inline java/mtsj/core/src/main/java/com/devonfw/application/mtsj/imagemanagement/common/api/Image.java:3]: Remove merge comments.

File: java/salesforcepoc/src/main/resources/application.properties [hunk 4]
```diff
@@ -4,11 +4,19 @@ server.host = localhost
 mts.backend.uri = http://localhost:8088/mythaistar/services/rest/register
 
 salesforce.auth.uri = https://login.salesforce.com
+<<<<<<< HEAD
```
Comment (mperezor19) [inline java/salesforcepoc/src/main/resources/application.properties:7]: Remove merge comments.

File: mrchecker/endtoend-test/pom.xml [hunk 5]
```diff
@@ -109,7 +109,11 @@
 
         <!-- Test tools -->
         <hamcrest.version>1.3</hamcrest.version>
+<<<<<<< HEAD
```
Comment (mperezor19) [inline mrchecker/endtoend-test/pom.xml:112]: Remove merge comments.

**Comment by monicalopezgris** [comment]
Open new request

**Pull request closed without merging**